ouroboros-network-framework/src/Ouroboros/Network/InboundGovernor.hs

```diff
@@ -176,10 +180,18 @@ inboundGovernor trTracer tracer serverControlChannel inboundIdleTimeout
             provenance
             connId
             csDataFlow
-            (Handle csMux muxBundle _ _)) -> do
+            (Handle csMux muxBundle _ versionData)) -> do
 
               traceWith tracer (TrNewConnection provenance connId)
 
+              -- Comunicate this new inbound connection to the Outbound Governor
+              when (provenance == Inbound) $
+                atomically $
+                  ControlChannel.writeMessage governorControlChannel
+                                              ( remoteAddress connId
+                                              , getPeerSharing versionData
```

Comment:
We should only pass duplex connections to the outbound governor.

Reply:
Docs for `NewConnection` say:

> A request to start mini-protocol bundle, either from the server or from
> connection manager after a duplex connection was negotiated.

Also the comment above says that:

> new connection has been announced by either accept loop or
> by connection manager (in which case the connection is in
> 'DuplexState').

So I guess this is already happening, right?

Reply:
This is implemented for passing outbound duplex connections to the inbound governor, but not when passing inbound connections to the outbound governor (the light peer sharing side).